nearcore/src/config.rs

```diff
@@ -1058,31 +1071,42 @@ pub fn get_genesis_url(chain_id: &String) -> String {
     )
 }
 
-pub fn download_genesis(url: &String, path: &PathBuf) {
-    info!(target: "near", "Downloading genesis file from: {} ...", url);
-
-    let url = url.clone();
-    let path = path.clone();
+pub fn get_config_url(chain_id: &String) -> String {
+    format!(
+        "https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/{}/config.json",
+        chain_id,
+    )
+}
 
+pub fn download_file(url: &String, path: &PathBuf, limit: usize) {
     actix::System::new().block_on(async move {
         let client = awc::Client::new();
-        let mut response =
-            client.get(url).send().await.expect("Unable to download the genesis file");
-
-        // IMPORTANT: limit specifies the maximum size of the genesis
-        // In case where the genesis is bigger than the specified limit Overflow Error is thrown
+        let mut response = client.get(url).send().await.expect("Unable to download the file");
+        // IMPORTANT: limit specifies the maximum size of the genesis or config file
+        // In case where the genesis or config file is bigger than the specified
+        // limit Overflow Error is thrown
         let body = response
             .body()
-            .limit(10_000_000_000)
+            .limit(limit)
             .await
-            .expect("Genesis file is bigger than 10GB. Please make the limit higher.");
-
-        std::fs::write(&path, &body).expect("Failed to create / write a config file.");
+            .expect("File is bigger than specified limit. Please make the limit higher.");
 
-        info!(target: "near", "Saved the config file to: {} ...", path.as_path().display());
+        std::fs::write(&path, &body).expect("Failed to create / write a file.");
     });
 }
 
+pub fn download_genesis(url: &String, path: &PathBuf) {
```

Comment:
Low importance Rust tip: don't use `&String` or `&PathBuf`, use `&str` and `&Path`:

https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#useless-types